tests/e2e_tests/cypress/e2e/basic-test/go-to-my-location-test.cy.js

```diff
@@ -0,0 +1,70 @@
+function lon2tile(lon, zoom) {
+    return Math.floor(((lon + 180) / 360) * Math.pow(2, zoom));
+}
+function lat2tile(lat, zoom) {
+    return Math.floor(
+        ((1 -
+            Math.log(Math.tan((lat * Math.PI) / 180) + 1 / Math.cos((lat * Math.PI) / 180)) /
+                Math.PI) /
+            2) *
+            Math.pow(2, zoom),
+    );
+}
```

Comment:
Can you add some comment as a context?
Is that translating lon and lan to some OSM (open street map) identifiers?

Now it's some strange math. :)